_posts/2018-12-24-snowplow-snowflake-loader-0.4.0-released/index.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title: Snowplow Snowflake Loader 0.4.0 released
+title-short: Snowflake Loader 0.4.0
+tags: [snowflake, storage, relational databases]
+author: Rostyslav
+category: Releases
+permalink: /blog/2018/12/24/snowplow-snowflake-loader-0.4.0-released/
+---
+
+We are pleased to announce version 0.4.0 of the [Snowplow Snowflake Loader][snowflake-loader-repo]! This release introduces optional cross-batch natural deduplication, as well as several other updates and bugfixes.
+
+Read on below the fold for:
+
+1. [Cross-batch deduplication](#crossbatch-dedupe)
+2. [New configuration options](#new-config)
+3. [Other changes](#other-changes)
+4. [Upgrading](#upgrading)
+5. [Getting help](#help)
+
+<!--more-->
+
+<h2 id="crossbatch-dedupe">1. Cross-batch deduplication</h2>
+
+Version 0.4.0 introduces [DynamoDB][dynamodb]-powered cross-batch natural deduplication, which works by extracting two identifier properties from an event (`event_id` and `event_fingerprint`), as well as `etl_tstamp` which identifies a single batch, then storing these properties in a DynamoDB table. Duplicate events with the same `event_id` and `event_fingerprint`, but a different `etl_tstamp`, found in storage are silently dropped from the Snowflake Transformer output.
```

Comment:
I think this *slightly* more technical details than it should be. Usually these sections are:

1. High-level overview of a problem (with link to more in-depth explanations, google "snowplow duplicated", there are lot of materials and previous Snowplow/Redshift pipeline posts)
2. The fact that we added it
3. Link to docs with in-depth setup guide 

Most of details covered here, should be in documentation instead. Also you can mention that there's new events manifest library now, that will be responsible for deduplication logic in our AWS pipelines.